__tests__/app.test.js

```diff
@@ -0,0 +1,88 @@
+const {
+    getTopics,
+  } = require("../db/helpers/utils");
+
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed")
+const testData = require("../db/data/test-data/index")
+
+beforeEach(() => {
+    return seed(testData)
+})
+
+afterAll(() => {
+  return db.end();
+});
+  
+  describe("getTopics", () => {
+    test("Responds with an array of topic objects, slug & description", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then((response) => {
+          expect(response.body.topics).toEqual(
+            [
+                {
+                  description: 'The man, the Mitch, the legend',
+                  slug: 'mitch'
+                },
+                {
+                  description: 'Not dogs',
+                  slug: 'cats'
+                },
+                {
+                  description: 'what books are made of',
+                  slug: 'paper'
+                }
+              ]
```

Comment:
please do this comparison in a more programmatic way, check the notes if you are unsure how to set this up, but you will effectively have 2 expect statements - one for the length of the array, and a second one where you forEach over the array and assert each element has the keys "slug" and "description", you can use the `expect.any(String)` syntax here to avoid hardcoding what the values of these topics are (we dont care what they are, only that the keys are there)